cheatsheets/Clickjacking_Defense_Cheat_Sheet.md

```diff
@@ -166,7 +166,7 @@ Some frame busting techniques navigate to the correct page by assigning a value
 **Victim frame busting code:**
 
 ```javascript
-if(top.location != self.locaton) {
+if(top.location != self.location) {
```

Comment:
This is actually a bug. As opposed to almost everything else here which is merely descriptive.